### ISSUE TITLE
how to ungroup shapes to svg ?

### ISSUE THREAD

**Issue by hellowac** [dbrnz/pptx2svg#5]
Thank you very much for your open source project, which has helped me a lot. I would like to ask how to ungroup the GroupShapes, then write them into svg format files?

tks very much!

**Comment by dbrnz** [comment]
You will need to modify the source code, say in your own branch or fork, to not create a new SVG group when a GroupShape is processed.

**Comment by hellowac** [comment]
tks

**Comment by hellowac** [comment]
When I did this, I encountered a new problem. The position of the svg is based on the absolute positioning of the upper left corner, and my requirement is that the svg is in the html page. I want to control the position of the svg in the page by myself, instead of the svg being fixed. Using the upper left corner as positioning, how to do it? I tried a lot of methods (including coordinate transformation/matrix transformation), but it didn't work, and I was very frustrated.
